test/org/zfin/datatransfer/ncbi/NCBIReleaseFetcherTest.java

```diff
@@ -0,0 +1,67 @@
+package org.zfin.datatransfer.ncbi;
```

Comment:
Add to a suite: 
New suite similar to DBTests for tests that make external web requests